src/AutoComplete/AutoComplete.js

```diff
@@ -325,7 +325,9 @@ class AutoComplete extends Component {
 
   handleBlur = (event) => {
     if (this.state.focusTextField && this.timerTouchTapCloseId === null) {
-      this.close();
+      setTimeout(() => {
```

Comment:
> It appears that handleBlur is executed before handleItemTouchTap

Interesting! In that case, should we defer the all `if` block into a `setTimeout`?
Shouldn't we also clear the timeout during the unmounting?